collatz-conjecture/collatz_conjecture.py

```diff
@@ -1,2 +1,17 @@
-def steps(number):
-    pass
+def steps(n):
+    count = 0
+    while n != 1:
+        print(count, n)
+        if (n % 2) == 0:
+            n = n/2
+            count += 1
+            
+        else:
+            n = 3*n + 1
+            count += 1
```

Comment:
One option is to have the "count += 1" line outside of the "if/else" statement.

`
if (n % 2) == 0:
            n = n/2
else:
            n = 3*n + 1

count += 1
`